databricks/koalas/typedef/string_typehints.py

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
This has to be a separate module due to the star import e.g.) `from numpy import *`.
It's in order to address the string literal type hint such as `"ks.Series"`. It's hacky but seems working at least.